src/components/10-atoms/progress-bar/README.md

```diff
@@ -0,0 +1,35 @@
+# Progress Bar
+
+A progress bar that can be used for when something is loading and when you can determind the progress.
+
+## Properties
+
+### Variant
+
+| Attribute   | Details                                                                       |
+| ----------- | ----------------------------------------------------------------------------- |
+| `small`     | Smaller version of the progress bar (boolean, default: false)                 |
+| `fullWidth` | Removes the `border-radius`so it can fit to an edge (boolean, default: false) |
+| `value`     | Sets the value (or progress) of the loader.                                   |
+| `max`       | Sets the max value of the progress bar (default: 100).                        |
+| `text`      | Sets the text below the progress bar.                                         |
+
+### small
+
+Smaller version of the progress bar (boolean, default: false)
+
+### fullWidth
+
+Removes the `border-radius`so it can fit to an edge (boolean, default: false)
```

Comment:
`Removes the `border-radius`, so it can fit edge to edge (boolean, default: false)`